src/main/riddl/Organization/organizationMessages.riddl

```diff
@@ -145,3 +145,39 @@ result OrganizationInfo is {
   orgId: OrganizationId,
   info: OrganizationContext.Info
 }
+query GetOrganizationParent is {orgId: OrganizationId}
+result OrganizationParent is {
+  orgId: OrganizationId,
+  parent: OrganizationId?
```

Comment:
Why is parent optional? If parent doesn't exist shouldn't the result be an error?

Reply:
If parent doesn't exist we are at top level (I assume since every state already has it as optional -may also be used to handle draft state option)

Reply:
Thinking out loud... What if the top level assumed itself as parent? 

This is in a result definition so I suppose it doesn't matter much. Just feels weird to have parent be optional.

Reply:
Per our conversation weeks ago, I am changing this to return self when there is no parent